ctapipe/image/extractor.py

```diff
@@ -1352,6 +1362,13 @@ def deconvolution_parameters(
 
     gains, shifts = [], []  # avg. gains and timing shifts of the deconvolved pulses
     for pz, ref_pulse_shape in zip(pzs, ref_pulse_shapes):
+        if time_profile_pdf:  # convolve ref_pulse_shape with time profile PDF
+            t = (
+                np.arange(ref_pulse_shape.size) - ref_pulse_shape.size / 2
+            ) * ref_sample_width_nsec
```

Comment:
[The pulse shape time is available as `subarray.tel[tel_id].camera.readout.ref](https://github.com/cta-observatory/ctapipe/blob/370b6b9a8464de0c3a5a06958ccc8b537d4b4f51/ctapipe/instrument/camera/readout.py#L118)